src/enum/cairn/ARMORS.ts

```diff
@@ -0,0 +1,16 @@
+import { ItemCairn } from "../../interfaces/CharacterCairn";
+
+const createArmor = (name: string, armor: string, bulky: boolean = false): Record<string, Partial<ItemCairn>> => {
+    const prettyArmorName = `${name} (${armor} Armor${bulky ? ', bulky': ''})`;
+    return {[name]: { name: prettyArmorName, type: 'armor', bulky: bulky, armor: armor }};
+}
+
+export const ARMORS: Record<string, Partial<ItemCairn>> = {
+    ... createArmor('Shield', '+1'),
+    ... createArmor('Helmet', '+1'),
+    ... createArmor('Gauntlets', '+1'),
```

Comment:
Gauntlets isn't part of starting gear

Reply:
You are right, my bad, Gauntlets should be a custom item.